webapp/channels/src/components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -172,7 +154,6 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
             isCloudTrial,
             hadPrevCloudTrial,
             isPaidSubscription,
-            minimumSKURequiredForFeature,
```

Comment:
(Comment not related to this specific line)

For the feature discovery pages, there are cases that are showing "Purchase License" and "Contact Sales" at the same time that both go to the Contact sales page. Should we just remove all Purchase License buttons in the feature discovery component? 
<img width="983" alt="Screenshot 2024-04-19 at 2 22 01 PM" src="https://github.com/mattermost/mattermost/assets/936315/240b34a6-d04e-46c8-9447-73cae60b5b3e">